docs/data/introduction/priority-support/priority-support.md

```diff
@@ -0,0 +1,67 @@
+- **4 hour pre-screening SLA**
+  Issues submitted through priority support channels undergo an initial review to validate, and reproduce the issue. This ensures your tickets have the right information for debugging.
+- **Issue Escalation**
+  Critical issues are escalated to senior engineers for focused attention and faster resolutions.
+
+For all the details, please check the [Service Level Agreement for Technical Support](https://mui.com/legal/technical-support-sla/).
+
+## Who benefits from priority support?
+
+Priority support is tailored for enterprise teams who rely on the MUI ecosystem at scale and want faster turnaround on bug reports, closer collaboration on issue triaging, and coverage across the entire component library.
+
+Priority support covers the following products:
+
+- MUI X
+  - Data Grid
+  - Date and Time Pickers
+  - Tree View
+  - Charts
+- MUI Core
+  - Material UI
+  - Base UI
+  - MUI System
+  - Pigment CSS
+  - Joy UI
+
+## How to buy priority support?
+
+Priority support is available as an add-on to the premium plan users. At $399/developer/year you can purchase it by reaching out to the support team. Both existing and new premium plan customers can buy it. In case, you are a pro plan user, you'll need to upgrade to the premium plan.
+
+Upon purchase you'll receive a support key that you need to validate when raising queries. You must keep it safe.
+
+## How to use priority support?
+
+### On GitHub
+
+- You'll go to either [mui-x](https://github.com/mui/mui-x) or [material-ui](https://github.com/mui/material-ui) repo and use the "**Priority support: SLA ⏰**" issue template to create a new issue invoking the Priority support SLA.
+- Once the issue is created it will have the following labels: [status: needs triage](https://github.com/mui/mui-x/labels/status%3A%20needs%20triage), [support: commercial](https://github.com/mui/mui-x/labels/support%3A%20commercial), [support: unknown](https://github.com/mui/mui-x/labels/support%3A%20unknown)
+- A GitHub action will prompt you to follow an external link leading to a validation app where you need to validate your support key.
+- Once your support key is successfully validated the labels in the issue will update to the following: [status: needs triage](https://github.com/mui/mui-x/labels/status%3A%20needs%20triage), [support: commercial](https://github.com/mui/mui-x/labels/support%3A%20commercial), [support: priority](https://github.com/mui/mui-x/labels/support%3A%20priority)
+- The GitHub action will update the previous comment with the following: `Thank you for verifying your support key 🔑, your SLA starts now.` Thus marking the start of the SLA.
+- You get a response from the team within the SLA.
+
+### Through email
+
+If you need to share **private information** you can [submit a request](https://support.mui.com/hc/en-us/requests/new?tf_360023797420=mui_x) or send an email to [x@mui.com](mailto:x@mui.com) with your support key.
```

Comment:
Just checking if this is the correct email ID we want to keep here?

Reply:
actually, let's remove the email.  Let's stay with the channels mentioned on: https://mui.com/legal/technical-support-sla/#support-channels